src/pages/groupComparison/GroupComparisonStore.ts

```diff
@@ -397,6 +399,26 @@ export default class GroupComparisonStore {
         invoke:()=>Promise.resolve(this.activeGroups.result![1])
     });
 
+    readonly referenceGenes = remoteData<ReferenceGenomeGene[]>({
+        await: ()=>[
+            this.studies,
+            this.mutationEnrichmentData,
+            this.copyNumberData
+        ],
+        invoke: async () => {
+            const mutGenes = this.mutationEnrichmentData.result!.map(
+                (a:AlterationEnrichment)=>a.hugoGeneSymbol.toUpperCase());
+            const cnvGenes = this.copyNumberData.result!.map(
+                (a:AlterationEnrichment)=>a.hugoGeneSymbol.toUpperCase());
+            return fetchReferenceGenomeGenes(this.studies.result[0].referenceGenome,
+                mutGenes.concat(cnvGenes));
+        },
+        onError:(err)=>{
```

Comment:
we should actually get rid of onError unless the error is specifically dealt with in your code.  Sorry, we are updating our practice around errors.

Reply:
getting rid of onError will allow global error handling to detect and show error message